sleap/nn/data/instance_cropping.py

```diff
@@ -42,12 +42,23 @@ def find_instance_crop_size(
     # Calculate crop size
     min_crop_size_no_pad = min_crop_size - padding
     max_length = 0.0
-    for inst in labels.user_instances:
-        pts = inst.points_array
-        pts *= input_scaling
-        max_length = np.maximum(max_length, np.nanmax(pts[:, 0]) - np.nanmin(pts[:, 0]))
-        max_length = np.maximum(max_length, np.nanmax(pts[:, 1]) - np.nanmin(pts[:, 1]))
-        max_length = np.maximum(max_length, min_crop_size_no_pad)
+    for lf in labels:
+        for inst in lf.user_instances:
```

Comment:
It would be faster to
```suggestion
    for lf in labels:
        for inst in lf:
            if isinstance(inst, sleap.PredictedInstance):
                continue
```
since
https://github.com/talmolab/sleap/blob/66d96ced50153e98b952176fbdb73043e105e397/sleap/instance.py#L1552-L1555